lib/commands/install.js

```diff
@@ -8,6 +8,8 @@ const { resolve, join } = require('path')
 const runScript = require('@npmcli/run-script')
 const pacote = require('pacote')
 const checks = require('npm-install-checks')
+const path = require('node:path')
+const rpj = require('read-package-json-fast')
```

Comment:
This module is being deprecated, we use `@npmcli/package-json` for all package.json interactions.

Reply:
I guess I still have to check that json object parsed from `package.json` is not an empty object since `@npmcli/package-json` doesn't check that for me?